.vscode/settings.json

```diff
@@ -8,6 +8,6 @@
     "omnisharp.enableEditorConfigSupport": true,
     "omnisharp.enableRoslynAnalyzers": true,
     "yaml.schemas": {
-         "https://raw.githubusercontent.com/microsoft/azure-pipelines-vscode/master/service-schema.json": "builds/azure-pipelines/**/*.yml"
```

Comment:
I don't know why this keeps getting modified, but 4 spaces should be the current setting and this is what I get when running format document so I'm just changing that now since it was always showing as changed otherwise...